python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -34,6 +34,10 @@ class LoadFileOperator(AstroSQLBaseOperator):
     :param columns_names_capitalization: determines whether to convert all columns to lowercase/uppercase
             in the resulting dataframe
     :param enable_native_fallback: Use enable_native_fallback=True to fall back to default transfer
+    :param skip_core_execution: Skips the core execution of the operator but run all ancillary operations. This is
+        useful for local run of tasks where subsequent tasks in the DAG might need the output of this operator like
+        XCOM results, but they do not want to actually run the core logic and hence not cause effects outside the
+        system.
```

Comment:
I'm not sure we should actually openly document this feature. This is not something I would want people using the python API to know about. A better option might be to keep it as a "hidden" feature and only run it if the argument happens to be in the kwargs.

Reply:
Yes. Removed it as a docstring param and accepting it as a kwarg. But popping it from kwargs and assigning it as an instance variable as the base class BaseOperator complains that invalid argument received upon initialtisation
```
 Traceback (most recent call last):
  File "/Users/pankajkoti/Library/Caches/pypoetry/virtualenvs/astro-sql-cli-r5JKmrdb-py3.8/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 411, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/Users/pankajkoti/Library/Caches/pypoetry/virtualenvs/astro-sql-cli-r5JKmrdb-py3.8/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 759, in __init__
    raise AirflowException(
airflow.exceptions.AirflowException: Invalid arguments were passed to LoadFileOperator (task_id: load_imdb_movies). Invalid arguments were:
**kwargs: {'skip_core_execution': True}
```